docs/source/index.rst

```diff
@@ -30,7 +30,7 @@ see the :ref:`Getting Started <getting started>` section.
 
 Starfish's documentation also contains instructions on :ref:`creating image-based transcriptomics
 processing pipelines <creating_an_image_processing_pipeline>`, tutorials on running
-starfish using the CLI, API, and workflow runner, and a worked example of contributing code to
+starfish using the API and workflow runner, and a worked example of contributing code to
```

Comment:
If we are removing the recipes code, then the workflow runner blurb is inaccurate.

```suggestion
starfish using the API, and a worked example of contributing code to
```